tests/test_layer_parameters.py

```diff
@@ -0,0 +1,191 @@
+import pytest
+from pyproj import CRS
+from city_metrix.layers import (
+    Layer,
+    Albedo,
+    AlosDSM,
+    AverageNetBuildingHeight,
+    BuiltUpHeight,
+    EsaWorldCover, EsaWorldCoverClass,
+    LandSurfaceTemperature,
+    NasaDEM,
+    NaturalAreas,
+    NdviSentinel2,
+    TreeCanopyHeight,
+    TreeCover,
+    UrbanLandUse,
+    WorldPop, OpenStreetMap, HighLandSurfaceTemperature
+)
+from tests.resources.bbox_constants import BBOX_BRA_LAURO_DE_FREITAS_1
+from tests.tools.general_tools import get_class_from_instance, get_class_default_spatial_resolution
+
+"""
+Evaluation of spatial_resolution property 
+To add a test for a scalable layer that has the spatial_resolution property:
+1. Add the class name to the city_metrix.layers import statement above
+2. Copy an existing test_*_spatial_resolution() test
+   a. rename for the new layer
+   b. specify a minimal class instance for the layer, not specifying the spatial_resolution attribute.
+"""
+
+COUNTRY_CODE_FOR_BBOX = 'BRA'
+BBOX = BBOX_BRA_LAURO_DE_FREITAS_1
+RESOLUTION_TOLERANCE = 1
+
+def test_albedo_spatial_resolution():
+    class_instance = Albedo()
+    doubled_default_resolution, actual_estimated_resolution =  evaluate_doubled_resolution_property(class_instance)
```

Comment:
Just to make sure I understand what you're testing here, you're doubling the resolution and then calculating the resolution separately, and checking they're close?

Reply:
That's correct.